client-sdk/ts-web/package-lock.json

```diff
@@ -4,6 +4,7 @@
     "requires": true,
     "packages": {
         "": {
+            "name": "ts-web",
```

Comment:
does it really add this? let me see if I can reproduce this locally

Reply:
reproduced locally, with npm 9.6.5. weird, `npm install` neither adds nor removes this. I guess let's have it here since `npm audit fix` really did put it there. feel free to remove whenever. it's not really a named package. it's just a workspace for holding workspaces.

Reply:
yeah :D even `npm ci` works in both cases

I'll remove it